scripts/verify_included_modules.py

```diff
@@ -172,7 +177,12 @@ def get_parser():
     return parser
 
 
-if __name__ == '__main__':
+def main():
+    """Main script to verify modules included."""
     parser = get_parser()
     args = parser.parse_args()
-    main(build_root=args.build_root)
+    verify_modules(build_root=args.build_root)
```

Comment:
Note moving the stuff from `__main__` (i.e. global scope) to `main()` was to appease `pylint`. I'm not sure when or why the OS-dependent path `'scripts/verify_included_modules.py'` ended up in `IGNORED_DIRECTORIES` in `run_pylint.py`.